iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -187,6 +163,9 @@ private async void ConnectionStatusChangeHandler(ConnectionStatus status, Connec
             {
                 case ConnectionStatus.Connected:
                     _logger.LogDebug("### The DeviceClient is CONNECTED; all operations will be carried out as normal.");
+                    // Call GetTwinOnConnectAsync() to retrieve twin values from the server once the connection status changes into Connected.

```

Comment:
nit: how about we add a new line between the log statement and the comment, for readability